src/assets/config.default.json

```diff
@@ -5,7 +5,7 @@
     "name": "app",
     "remote_url": "http://localhost:4200/db/",
     "timeout": 60000,
-    "useTemporaryDatabase": true
+    "useTemporaryDatabase": false
```

Comment:
Please don't change the config.default.json (at least if there is no special reason to change it for every developer). Copy this file as assets/config.json and change your settings there (see https://github.com/Aam-Digital/ndb-core/blob/master/README.md#configuration)